### PR TITLE
Avoid camera sensors for GUI-controlled agents

### PR DESCRIPTION
**Pull request by rpartsey**
## Motivation and Context

Avoid camera sensors for GUI-controlled agents.

## How Has This Been Tested
Running Sandbox App locally (MacBook Pro 2021).

## Types of changes

- **Development**

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes if required.

### CODE REVIEW

File: examples/siro_sandbox/sandbox_app.py [hunk 1]
```diff
@@ -1089,6 +1100,14 @@ def parse_debug_third_person(args, framebuffer_size):
 
     driver = SandboxDriver(args, config, gui_app_wrapper.get_sim_input())
 
+    # sanity check if there are no agents with camera sensors
+    # if (
+    #     len(config.habitat.simulator.agents) == 1
+    #     and args.gui_controlled_agent_index is not None
+    # ):
+    #     assert driver.get_sim().createRenderer == False
```
Comment (rpartsey) [inline examples/siro_sandbox/sandbox_app.py:1108]: AttributeError: 'RearrangeSim' object has no attribute 'createRenderer'.

File: examples/siro_sandbox/sandbox_app.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
     "data/humanoids/humanoid_data/walking_motion_processed_smplx.pkl"
 )
 
-DEFAULT_CFG = "benchmark/rearrange/rearrange_easy_human_and_fetch.yaml"
+DEFAULT_CFG = "experiments_hab3/pop_play_kinematic_oracle_humanoid_spot.yaml"
```
Comment (rpartsey) [inline examples/siro_sandbox/sandbox_app.py:48]: Post https://github.com/facebookresearch/habitat-lab/pull/1370 PR default param fix.

**Review by eundersander — APPROVED** [review]
I think we can find a way to check whether the renderer got created. I posted an idea. Aside from that, LGTM!

File: examples/siro_sandbox/sandbox_app.py (hunk 1 above)
Comment (rpartsey) [reply examples/siro_sandbox/sandbox_app.py:1108]: Thank you!

Here are the values:
Screenshot 2023-05-26 at 12 09 36

**Pull request merged**